src/components/v2/V2Create/tabs/ReviewDeployTab/index.tsx

```diff
@@ -3,27 +3,37 @@ import { Trans } from '@lingui/macro'
 import { useContext } from 'react'
 
 import { ThemeContext } from 'contexts/themeContext'
+import { Space } from 'antd'
+import { Gutter } from 'antd/lib/grid/row'
+import useMobile from 'hooks/Mobile'
 
 import DeployProjectButton from './DeployProjectButton'
-import ProjectPreview from '../../ProjectPreview'
+import ProjectDetailsSection from './ProjectDetailsSection'
+import FundingSection from './FundingSection'
+
+export const rowGutter: [Gutter, Gutter] = [40, 30]
 
 export default function ReviewDeployTab() {
   const {
     theme: { colors },
   } = useContext(ThemeContext)
+  const isMobile = useMobile()
   return (
     <div>
-      <h1 style={{ fontSize: '1.5rem' }}>
+      <h1 style={{ fontSize: !isMobile ? '1.5rem' : '1.3rem' }}>
         <Trans>Review project configuration</Trans>
       </h1>
       <div
         style={{
-          padding: '3rem',
+          padding: !isMobile ? '3rem' : '2rem',
           border: '1px solid' + colors.stroke.tertiary,
```

Comment:
I think we can just get rid of the padding and border now.

We might still need some bottom margin on the `Review project configuration` h1 tag though - see what you think

Reply:
Yep done. This how it looks now on desktop/mobile. (Note: still needs a bit of padding on mobile)

<img width="1273" alt="Screen Shot 2022-05-02 at 8 12 09 pm" src="https://user-images.githubusercontent.com/96150256/166310701-ee2bee4d-b377-4a95-87aa-cfca80359648.png">

<img width="308" alt="Screen Shot 2022-05-02 at 8 15 44 pm" src="https://user-images.githubusercontent.com/96150256/166310708-6d14f187-2a8f-4f19-8745-bc46e6b775d7.png">